SU2_CFD/include/fluid/CDiffusivityModel.hpp

```diff
@@ -0,0 +1,51 @@
+/*!
+ * \file CDiffusivityModel.hpp
```

Comment:
This file is still not used.

Reply:
Yes it is still not used, I have to remove it from this pull request. Thank you so much for the feedback.